internal/cli/runner_profile_set.go

```diff
@@ -250,6 +260,13 @@ func (c *RunnerProfileSetCommand) Flags() *flag.Sets {
 			Usage: "Indicates that this on-demand runner should be used by any project that doesn't " +
 				"otherwise specify its own on-demand runner.",
 		})
+
+		f.StringVar(&flag.StringVar{
+			Name:    "target-runner-id",
+			Target:  &c.flagTargetRunnerId,
+			Default: "",
+			Usage:   "ID of the on-demand runner to target for the profile.",
```

Comment:
I think this is the ID of the NOT on-demand runner. I don't know what we consistently call those publicly, something like "static" runner.

Reply:
Oh, good catch, meant to say "remote" runner. Though this was a point in the RFC too, we refer to these specific remote runners as "static" internally, but not in any external facing documentation. I'm all for using "static" runners though, if we want to start.